src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+/*
+  eslint max-len: ["error", { "ignoreStrings": true }]
+*/
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle',
```

Comment:
Create a separate `enum` for shape type. This type should be independent and not related to `Figure` interface